requirements.txt

```diff
@@ -16,9 +16,9 @@ colorama==0.4.5 \
     --hash=sha256:854bf444933e37f5824ae7bfc1e98d5bce2ebe4160d46b5edf346a89358e99da \
     --hash=sha256:e6c6b4334fc50988a639d9b98aa429a0b57da6e17b9a44f0451f930b6967b7a4
     # via -r requirements.in
-django==4.0.2 \
-    --hash=sha256:110fb58fb12eca59e072ad59fc42d771cd642dd7a2f2416582aa9da7a8ef954a \
-    --hash=sha256:996495c58bff749232426c88726d8cd38d24c94d7c1d80835aafffa9bc52985a
+Django==4.0.6 \
```

Comment:
We will forever be fighting with the test and renovate, pip compile does not output a capital `D`, no matter what the input, and renovate changes it to an uppercase letter for some reason. Seems like a bug in renovate to me, it shouldn't be changing the dependency package names.